src/views/email.handlebars

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
prettify a little bit

Reply:
should I make the email template prettier? or code itself